plugin.js

```diff
@@ -107,8 +107,14 @@ CKEDITOR.plugins.add('uploadcare', {
                 uc.jQuery.ajax({
                   url: '/images/uploadcare',
                   method: 'POST',
-                  contentType: 'application/json;charset=utf-8',
+                  contentType: 'application/json',
                   dataType: 'json',
+                  beforeSend: function (xhr) {
+                    var xsrf = CKEDITOR.tools.getCookie('XSRF-TOKEN');
```

Comment:
is `CKEDITOR.tools` guaranteed to be there? otherwise, maybe something like `var xsrf = CKEDITOR && CKEDITOR.tools && CKEDITOR.tools.getCookie('XSRF-TOKEN');`

Reply:
seems like it's part of the core ckeditor stuff, so it's probably ok.